grpc_codegen/avs_grpc_pb.js

```diff
@@ -344,6 +377,4 @@ ping: {
   },
 };
 
-const AggregatorClient = grpc.makeGenericClientConstructor(AggregatorService);
-
-export { AggregatorService, AggregatorClient };
```

Comment:
The last line is needed after the `npm run gen-protoc` command, otherwise, these variables are not exported.

Here’s my instruction in the README.md
> Important: the last line of the `grpc_codegen/avs_pb.js` needs to be manually added after the `gen-protoc` command. These type definitions must be exported; otherwise they will be undefined in the SDK. For example: `export const { Task, CreateTaskReq, CreateTaskResp, GetKeyReq, KeyResp, UpdateChecksReq, UpdateChecksResp, AddressResp, AddressRequest } = proto.aggregator;`

Reply:
Without these export, the `CreateTaskReq, CreateTaskResp` will return undefined.

Reply:
We shouldn't need to do that. I think there is something else had masked that issue when you test them out. 

This is run directly in node console, without needing that extra line. If we had to do that, it really mean there is an issue with the grpc codegen cli and we should report to upstream.

![CleanShot 2024-11-20 at 01 39 48](https://github.com/user-attachments/assets/e43cdc74-7ef7-43ab-bdd9-124c55a9c217)

```
 tail -n 10 grpc_codegen/avs_pb.js
};

/**
 * @enum {number}
 */
proto.aggregator.CustomCodeType = {
  JAVASCRIPT: 0
};

goog.object.extend(exports, proto.aggregator);
```

Reply:
yeah, the cli generation caused those undefined problems. So is the auto-generated code working now?